addons/sale/models/sale_order.py

```diff
@@ -12,25 +12,264 @@
 
 
 class SaleOrder(models.Model):
-    _name = "sale.order"
+    _name = 'sale.order'
```

Comment:
why?

Reply:
For our team guideline saying that user-shown strings are defined in double quotes, and tech strings are specified as single quotes :D.